openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1246,9 +1246,9 @@ This specification defines the following Credential Issuer Metadata:
 * `display`: OPTIONAL. An array of objects, where each object contains display properties of a Credential Issuer for a certain language. Below is a non-exhaustive list of valid parameters that MAY be included:
   * `name`: OPTIONAL. String value of a display name for the Credential Issuer.
   * `locale`: OPTIONAL. String value that identifies the language of this object represented as a language tag taken from values defined in BCP47 [@!RFC5646]. There MUST be only one object for each language identifier.
-* `credentials_supported`: REQUIRED. An object that describes specifics of the Credential that the Credential Issuer supports issuance of. This object contains a list of name/value pairs, where each name is a unique identifier of the supported Credential being described. This identifier is used in the Credential Offer as defined in (#credential_offer_parameters) to communicate to the Wallet which Credential is being offered. The value is an object that contains metadata about specific Credential and contains the following parameters defined by this specification:
+* `credential_configurations_supported`: REQUIRED. An object that describes specifics of the Credential that the Credential Issuer supports issuance of. This object contains a list of name/value pairs, where each name is a unique identifier of the supported Credential being described. This identifier is used in the Credential Offer as defined in (#credential_offer_parameters) to communicate to the Wallet which Credential is being offered. The value is an object that contains metadata about specific Credential and contains the following parameters defined by this specification:
```

Comment:
```suggestion
* `credential_configurations_supported`: REQUIRED. An object detailing the specifics of the Credential that the Credential Issuer is capable of issuing. This object comprises a list of name/value pairs, where each name uniquely identifies the supported Credential being described. This identifier is utilized in the Credential Offer, as outlined in (#credential_offer_parameters), to inform the Wallet about the offered Credential. The value is an object that contains metadata about a specific Credential and includes the following parameters defined by this specification:
```

Reply:
please do the bikeshedding in PR https://github.com/openid/OpenID4VCI/pull/155. this PR is strictly meant to change the parameter names only.